pyaedt/modeler/cad/Primitives.py

```diff
@@ -4924,6 +4924,70 @@ def import_spaceclaim_document(self, input_file):
         self.refresh_all_ids()
         return True
 
+    def discovery_import(self, input_file):
```

Comment:
@IreneWoyna I'd keep the method name as in AEDT: add_discovery_link

Reply:
Ok, that's fine. I changed the name because I didn't implement the actual "dynamic link", only static import (because it works better)